merlin/models/tf/core.py

```diff
@@ -2274,6 +2367,33 @@ def get_config(self):
         return {"block": tf.keras.utils.serialize_keras_object(self.block)}
 
 
+class MetricsComputeCallback(tf.keras.callbacks.Callback):
+    def __init__(self, train_metrics_steps=1, **kwargs):
```

Comment:
Maybe we should log how often we update the metrics here.

Reply:
When we fit() with keras, we can observe that metrics are only updated each N steps, keeping the last result for the other steps. That way we can verify that the metrics are really being computed each N steps.